CDP4CrossViewEditor/ViewModels/ThingSelectorViewModel.cs

```diff
@@ -214,36 +177,18 @@ private void AddSubscriptions()
             this.MoveItemsUp.Subscribe(_ => this.ExecuteMoveUp());
 
             this.MoveItemsDown = ReactiveCommand.Create();
-            this.MoveItemsDown.Subscribe(_ => this.ExecuteMoveItemsDown());
+            this.MoveItemsDown.Subscribe(_ => this.ExecuteMoveDown());
 
             this.ClearItems = ReactiveCommand.Create();
-            this.ClearItems.Subscribe(_ => this.ExecuteClearItems());
+            this.ClearItems.Subscribe(_ => this.ExecuteClear());
 
             this.SortItems = ReactiveCommand.Create();
-            this.SortItems.Subscribe(_ => this.ExecuteSortItems());
-        }
-
-        /// <summary>
-        /// Executes move to source command <see cref="MoveItemsToSource"/>
-        /// Things will be moved back to selection source
-        /// </summary>
-        private void ExecuteMoveToSource()
-        {
-            // TODO #625: Implement ordering and sorting support for selected elements
-        }
-
-        /// <summary>
-        /// Executes move to target command <see cref="MoveItemsToTarget"/>
-        /// Things will be selected for Cross View Editor table/report
-        /// </summary>
-        private void ExecuteMoveToTarget()
-        {
-            // TODO #625: Implement ordering and sorting support for selected elements
+            this.SortItems.Subscribe(_ => this.ExecuteSort());
         }
 
         /// <summary>
         /// Executes move items command <see cref="MoveItemsUp"/>
-        /// Move selected <see cref="Thing"/> one position up inside selection list
+        /// ExecuteMove selected <see cref="Thing"/> one position up inside selection list
         /// </summary>
         private void ExecuteMoveUp()
```

Comment:
Q3: Do we still need this method for manual moving since we're now using grids?

Reply:
No